tools/reaction_analyzer/README.md

```diff
@@ -0,0 +1,119 @@
+# Reaction Analyzer
+
+## Description
+
+The main purpose of the reaction analyzer package is measuring reaction times of the nodes by listening the
+pre-determined topics. It can be used for measuring the reaction time of the perception pipeline, planning pipeline, and
+control pipeline. To be able to measure both control outputs and perception outputs, it is necessary to divide the node
+into two running_mode: `planning_control` and `perception_planning`.
+
```

Comment:
Maybe you can add some images of node diagram to show how the analyzer calculates the reaction time.